tests/assets/acceptance/session_management/1Daqmx2ChannelGroup2Pin1Site.pinmap

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PinMap xmlns="http://www.ni.com/TestStand/SemiconductorModule/PinMap.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schemaVersion="1.1">
+	<Instruments>
+		<NIDAQmxTask name="Dev1a" taskType="AnalogInput" channelList="Dev1/ai0, Dev1/ai1" />
+		<NIDAQmxTask name="Dev1b" taskType="AnalogInput" channelList="Dev1/ai2, Dev1/ai3" />
```

Comment:
This will not work because X Series MIOs only support running one AI task at a time. If you want multiple AI tasks with X Series, you need multiple devices (e.g. Dev1 and Dev2). 

It only *seemed* to work because you were running the tasks one at a time.

This will require updating the simulated device configuration on the self-hosted GitHub runner. See the measlinkbot verison of nisimdev_simulate.sls in ni-central.